bug-bounty-preview-scopes.md

```diff
@@ -0,0 +1,33 @@
+# .NET Bounty Scope
+
+## Introduction
+
+The [.NET bug bounty](https://www.microsoft.com/en-us/msrc/bounty-dot-net-core) covers current released versions of .NET Core, ASP.NET Core and .NET, as well as the latest beta or release candidate of the upcoming version.
+
+.NET has the concept of preview features which ship in the current version, or in nightly builds of the upcoming version. Preview features are not enabled by default, but their goal is to be enabled in the next major release. Preview features are now considered in scope for the bug bounty if they are listed in the table below.
+
+Bugs against preview features in the current release must be demonstrated against the latest current release.
+
+Bugs against preview features in nightly builds of the next version must be demonstrated against the build number listed in the table below, or subsequent builds. Preview features in nightly builds may fall out of scope while bugs are addressed or the removal of the feature from the upcoming version.
```

Comment:
> Bugs against preview features in nightly builds of the next version must be demonstrated against the build number listed in the table below, or subsequent builds.

How often do you plan to update this table? Also, maybe I'm missing something, but why are the rules for nightlies different from current releases (latest only)?

Reply:
Because nightlies have been excluded because they won't have undergone security reviews and contain half-baked stuff there for testing. This now includes features which we consider complete and ready for beating up, even if they're in nightlies, but limits the scope to those nightlies, keeping legal happy :)